aesm-client/src/lib.rs

```diff
@@ -150,12 +162,16 @@ impl QuoteResult {
     }
 }
 
-#[derive(Default, Debug, Clone)]
+// TODO: Find a better alternative to removing Clone and Default trait
+#[cfg_attr(not(target_env = "sgx"), derive(Default,Clone))]
```

Comment:
Why is clone conditional? You should manually implement `Clone` for imp::AesmClient on SGX since that's guaranteed to work